src/sys/memfd.rs

```diff
@@ -102,6 +115,7 @@ pub fn memfd_create<P: NixPath + ?Sized>(
                     // If the OS is Linux, gnu and musl expose a memfd_create symbol but not uclibc
```

Comment:
Let's give this comment an update:


```suggestion
                    // If the OS is Linux, gnu/musl/ohos expose a memfd_create symbol but not uclibc
```